Solving guidance page (#23777)
sympy/sympy: doc/src/guides/solving/index.md

linear equations, nonlinear equations, matrix problems, inequalities,
Diophantine equations, and evaluate integrals. SymPy can also solve numerically.

> You can refer to [solving guidance](solving-guidance.md) applicable to many
> types of solving.

Reviewer: ```suggestion
Refer to the document [here](solving-guidance.md) for guidance applicable to many
types of solving tasks.
```
